src/spectral/rulesets/.defaultsForSpectral.yaml

```diff
@@ -244,3 +244,17 @@ rules:
       - $.servers[*][variables][*][default]
     then:
       function: truthy
+  # ensure that examples name does not contain space
+  examples-name-should-not-contain-space:
```

Comment:
I'm not sure this name is good enough. I'm open for suggestions.

Reply:
We used to name rules after the problem it catches - something like `examples-name-contains-space`. We might use a different convention now. @barrett-schonefeld do you have an opinion?

Reply:
I think `examples-name-contains-space` is a good choice here.